skrub/_agg_pandas.py

```diff
@@ -0,0 +1,142 @@
+import re
+from itertools import product
+
+import numpy as np
+import pandas as pd
+
+
+def aggregate(
+    table,
+    cols_to_join,
+    cols_to_agg,
+    num_ops,
+    categ_ops,
+    suffix,
+):
```

Comment:
We need a fairly complete docstring on this: future developers will be trying to use it and need to understand all arguments.

Also: can any of these arguments take default values (become keyword arguments), to facilitate usage?